scanner/src/main/kotlin/provenance/PackageProvenanceResolver.kt

```diff
@@ -209,13 +209,16 @@ class DefaultPackageProvenanceResolver(
                 vcs.getRevisionCandidates(workingTree, pkg, allowMovingRevisions = true)
             }.getOrDefault(emptySet())
 
-            if (revisionCandidates.isEmpty()) {
-                val message = "Could not find any revision candidates for package '${pkg.id.toCoordinates()}' with " +
-                        "VCS ${pkg.vcsProcessed}."
+            val errorMessages = mutableListOf<String>()
 
-                storage.putProvenance(pkg.id, pkg.vcsProcessed, UnresolvedPackageProvenance(message))
+            fun addAndLogError(message: String) {
+                logger.info { message }
```

Comment:
Shouldn't errors also be logged at `error` level?

Reply:
I think "info" is the right log level for these messages because issues during provenance resolution are expected and often do not lead to an error but just to trying another candidate. But I have renamed the function to accommodate for that.